### PR TITLE
Sửa lại phần hiển thị trên lịch làm việc

### PR DESCRIPTION
**Pull request by phanvantan73**

### CODE REVIEW

File: app/Repositories/LocationRepository.php [hunk 1]
```diff
@@ -137,12 +137,12 @@ public function analystic(
                 'title' => $title . ' ' . $count,
                 'className' => $className,
                 'url' => route(
-                    'calendar.location.detail_location',
-                    [
-                        'id' => $location->id,
-                        'date' => $date,
-                    ]
-                ) . '?shift=' . $filter['shift'],
+                        'calendar.location.detail_location',
```
Comment (framgiaciv3) [inline app/Repositories/LocationRepository.php]: **Reporter**: CHECKSTYLE</br>**Rule**: PSR2.Methods.FunctionCallSignature.Indent</br>**Severity**: error</br>**File**: app/Repositories/LocationRepository.php</br>**Line**: 140</br>Multi-line function call not indented correctly; expected 20 spaces but found 24

File: app/Repositories/LocationRepository.php [hunk 2]
```diff
@@ -137,12 +137,12 @@ public function analystic(
...
+                        [
```
Comment (framgiaciv3) [inline app/Repositories/LocationRepository.php]: **Reporter**: CHECKSTYLE</br>**Rule**: PSR2.Methods.FunctionCallSignature.Indent</br>**Severity**: error</br>**File**: app/Repositories/LocationRepository.php</br>**Line**: 141</br>Multi-line function call not indented correctly; expected 20 spaces but found 24

File: app/Repositories/LocationRepository.php [hunk 3]
```diff
@@ -137,12 +137,12 @@ public function analystic(
...
+                            'id' => $location->id,
+                            'date' => $date,
+                        ]
+                    ) . '?shift=' . $filter['shift'],
```
Comment (framgiaciv3) [inline app/Repositories/LocationRepository.php]: **Reporter**: CHECKSTYLE</br>**Rule**: PSR2.Methods.FunctionCallSignature.Indent</br>**Severity**: error</br>**File**: app/Repositories/LocationRepository.php</br>**Line**: 145</br>Multi-line function call not indented correctly; expected 16 spaces but found 20

**Comment by phanvantan73** [comment]
https://edu-redmine.framgia.vn/issues/3453

File: resources/views/admin/calendar/detailLocation.blade.php [hunk 4]
```diff
@@ -5,107 +5,105 @@
...
-                            <div class="m-section__content">
-                                <table class="table m-table m-table--head-bg-success">
-                                    <thead>
+                        {!! Form::open(['class' => 'row mb-3', 'method' => 'GET']) !!}
+                        <div class="col-md-4 offset-md-2">
+                            {!! Form::select('shift', config('site.shift_filter'), request('shift'), ['class' => 'form-control m-input m-input--square', 'id' => 'program']) !!}
+                        </div>
+                        <div class="col-md-4">
+                            {!! Form::select(
+                                    'program_id',
```
Comment (tranducquoc) [inline resources/views/admin/calendar/detailLocation.blade.php]: indent

File: resources/views/admin/calendar/detailLocation.blade.php [hunk 5]
```diff
@@ -5,107 +5,105 @@
...
+                                                            <div class="m-card-user__pic">
+                                                                <div class="m-card-user__no-photo m--bg-fill-danger">
                                                                 <span>
                                                                     {!! Html::image($user->avatarUser) !!}
                                                                 </span>
+                                                                </div>
+                                                            </div>
+                                                            <div class="m-card-user__details">
+                                                            <span class="m-card-user__name">
+                                                                <a href="{{ url('admin/schedule/users/' . $user->id) }}">{{ $user->name }}</a>
```
Comment (tranducquoc) [inline resources/views/admin/calendar/detailLocation.blade.php]: dùng routes đi em

File: resources/views/admin/work_schedules/view_location.blade.php [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+            @forelse($locationList as $item)
+                <div class="col-lg-6">
+                    <div class="m-portlet m-portlet--success m-portlet--head-solid-bg m-portlet--head-sm" m-portlet="true" id="m_portlet_tools_2">
+                        <div class="m-portlet__head">
+                            <div class="m-portlet__head-caption">
+                                <div class="m-portlet__head-title">
+                            <span class="m-portlet__head-icon">
+                                <i class="fa fa-warehouse"></i>
+                            </span>
+                                    <h3 class="m-portlet__head-text">
```
Comment (tranducquoc) [inline resources/views/admin/work_schedules/view_location.blade.php]: indent

File: app/Repositories/LocationRepository.php [hunk 7]
```diff
@@ -192,4 +192,59 @@ public function getLocationDetail($filter)
...
+    }
+
+    public function getTotalUsersInDay($location, $filter)
+    {
+        $data = $location->workSchedules()
+            ->select(DB::raw('COUNT(user_id) as total, date as start'))
+            ->whereBetween('date', [$filter['start'], $filter['end']]);
+
+        if (array_key_exists('program_id', $filter) && $filter['program_id']) {
+            $getUserByProGram = DB::table('users')->where('program_id', $filter['program_id'])->pluck('id');
```
Comment (BunnyPi04) [inline app/Repositories/LocationRepository.php]: sao ko dùng eloquent nhỉ?

File: resources/views/admin/calendar/detailLocation.blade.php [hunk 8]
```diff
@@ -5,107 +5,102 @@
...
+                                                    <td>
+                                                        <span class="btn btn-primary">{{ $user->getShiftByDate($date) }}</span>
+                                                    </td>
+                                                </tr>
+                                            @empty
+                                                @include('admin.components.alert', ['type' => 'warning', 'message' => __('Have no data!')])
+                                            @endforelse
+                                        </tbody>
+                                    </table>
+                                    {{ $data->links() }}
```
Comment (BunnyPi04) [inline resources/views/admin/calendar/detailLocation.blade.php]: đoạn này nếu được thì xử lý ajax phân trang nhé

**Pull request merged**